crates/sui-indexer/migrations/pg/2024-09-12-213234_watermarks/up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE watermarks
+(
+    -- The table governed by this watermark, i.e `epochs`, `checkpoints`, `transactions`.
+    entity                      TEXT          NOT NULL,
+    -- Inclusive upper epoch bound for this entity's data. Committer updates this field. Pruner uses
+    -- this to determine if pruning is necessary based on the retention policy.
+    epoch_hi                    BIGINT        NOT NULL,
```

Comment:
nit: Following the convention we set out during the work on transactions scan limits etc, this should be `epoch_hi_inclusive` to call attention to the fact that it's an upperbound, but is inclusive -- would it be too late to change that now?

Similar comments for the other inclusive upperbounds in this table.